.travis.yml

```diff
@@ -1,10 +1,8 @@
-dist: trusty
+dist: xenial
 language: php
 
 php:
-  - 7.1
-  - 7.2
-  - 7.3
+  - 7.4
```

Comment:
Remove `php-7.1`, `php-7.2` and `php-7.3` version tests during Travis CI building.